cmd/run.go

```diff
@@ -37,6 +40,10 @@ func init() {
 	runCmd.PersistentFlags().StringArrayVarP(&taskRequest.Mounts, "mount", "m", []string{}, "bind-mount a volume the task")
 	runCmd.PersistentFlags().StringArrayVarP(&taskRequest.Secrets, "secret", "s", []string{}, "Add existing secret to task")
 
+	runCmd.PersistentFlags().StringVarP(&taskRequest.ReportUrl, "report-url", "p", "", "Report URL for Helio")
+	runCmd.PersistentFlags().StringVarP(&taskRequest.HeartbeatUrl, "heartbeat-url", "y", "", "Heartbeat URL for Helio")
```

Comment:
you don't need a short name for a flag if you don't need one. why `y` for heartbeat-url`? :D